praetorian_cli/runners/local.py

```
@@ -179,6 +199,18 @@ def install_tool(tool_name: str, force=False) -> str:
     # Make executable
     os.chmod(binary_path, 0o755)
 
+    # Record version
+    try:
+        from praetorian_cli.registry import get_registry
+        ver_result = subprocess.run(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)